agb/src/display/object.rs

```diff
@@ -1193,6 +1193,42 @@ enum ColourMode {
     Eight,
 }
 
+/// The parameters used for the PPU's affine transformation function
+/// that can apply to objects and background layers in modes 1 and 2.
+/// This can be obtained from X/Y scale and rotation angle with
+/// [`agb::syscall::affine_matrix`].
+#[derive(Copy, Clone, Debug, PartialEq, Eq)]
+#[repr(C, packed)]
```

Comment:
```suggestion
#[repr(C, packed(4))]
```

Should be aligned to 4 bytes (efficiency + correctness).